importer/views.py

```diff
@@ -0,0 +1,43 @@
+"""
+Views for the importer app.
+"""
+
+from django.shortcuts import render
+from django.contrib.auth.models import User
+
+from learningresources.models import Repository, Course, LearningResource
+from .forms import UploadForm
+
+
+def status(request):
+    """
+    Show what's currently in the system.
+    """
+    counts = {
+        "repos": Repository.objects.count(),
+        "courses": Course.objects.count(),
+        "loxes": LearningResource.objects.count(),
+    }
+    return render(
+        request,
+        "status.html",
+        counts,
+    )
+
+
+def upload(request):
+    """
+    Upload a OLX archive.
+    """
+    form = UploadForm()
+    if request.user.is_anonymous():
+        request.user, _ = User.objects.get_or_create(username="dirty_hack")
```

Comment:
I didn't expect that this was going live anywhere before the authentication kicked in. Is it?

I added `TODO` comments, but then it fails tox, so I took them out.

Also, as for large tarballs, I assumed that the server (nginx?) will have max sizes to protect us from people uploading huge files. I know we can set a max size in Django also, but then you have to maintain it in two places. So, just a note: The server software should have a setting to limit the size.